### PR TITLE
MF-63: Styleguide documentation -- show React JSX

### PR DESCRIPTION
**Pull request by fatmali**
Screen Recording 2019-09-25 at 5 08 20 PM

### CODE REVIEW

File: .storybook/custom-decorators/tabs.decorator.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+function getTabs() {
+  return document.querySelectorAll(".tab");
+}
+
+function getPanels() {
+  return document.querySelectorAll(".panel");
+}
+
+function saveLastViewedLang(lang) {
+  localStorage.setItem(lastViewedLangKey, lang);
```
Comment (joeldenning) [inline .storybook/custom-decorators/tabs.decorator.js]: 👍 👍 i like this a lot

File: .storybook/theme.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { create } from "@storybook/theming";
+import logo from "../logo/openmrs_logo.png";
```
Comment (joeldenning) [inline .storybook/theme.js]: Could we reuse the logo svg icon for this?

File: .storybook/custom-decorators/tabs.decorator.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+function showLastViewedLangTab() {
+  const lastViewedLang = getLastViewedLang() || "html";
+  const tabs = document.querySelectorAll(`div[lang=${lastViewedLang}]`);
+  const panels = document.querySelectorAll(`.${lastViewedLang}`);
+  showAll(tabs);
+  showAll(panels);
+}
+
+export const withTabs = storyFn => {
+  useEffect(() => {
```
Comment (joeldenning) [inline .storybook/custom-decorators/tabs.decorator.js]: Oo I didn't know that this existed. 👍 👍

File: .storybook/custom-decorators/tabs.decorator.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import { useEffect } from "@storybook/client-api";
+
+const lastViewedLangKey = "omrs-storybook:preferred-lang";
+
+function getLastViewedLang() {
+  return localStorage.getItem(lastViewedLangKey);
+}
+
+function getTabs() {
+  return document.querySelectorAll(".tab");
```
Comment (joeldenning) [inline .storybook/custom-decorators/tabs.decorator.js]: So we switch all the tabs at once, across all examples? Not just the tab you clicked on.

I guess that's probably a pretty good ux.

File: src/story-helpers.css [hunk 5]
```diff
@@ -11,3 +11,36 @@ pre {
...
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+  width: 100%;
+  overflow: hidden;
+  line-height: 36px;
+}
+.examples .tabs .tab {
+  float: left;
+  background: white;
+  color: #777777;
+  height: 31px;
```
Comment (joeldenning) [inline src/story-helpers.css]: why did you switch from rem on line 17 to px here and elsewhere?

File: .storybook/custom-decorators/tabs.decorator.js (hunk 4 above)
Comment (fatmali) [reply .storybook/custom-decorators/tabs.decorator.js]: that's right, switch all tabs at once to the syntax of the tab clicked.

File: .storybook/custom-decorators/tabs.decorator.js (hunk 3 above)
Comment (fatmali) [reply .storybook/custom-decorators/tabs.decorator.js]: released with version v5.2, had to dig really deep to find this.

File: .storybook/theme.js (hunk 2 above)
Comment (fatmali) [reply .storybook/theme.js]: to use the svg, it requires the width and height to be set, we haven't set it for the logo svg icons.

**Comment by fatmali** [comment]
@joeldenning updated the PR as per the review.

**Pull request merged**